### DISCUSSION TITLE
Plank Desktop error

### DISCUSSION THREAD

**Discussion by alemstrom** [elementary/triage#648] [Q&A]
Getting repetitive errors of the following:

plank.desktop[2873]: [2919:2925:1207/134636.419677:ERROR:ssl_client_socket_impl.cc(878)] handshake failed; returned -1, SSL error code 1, net_error -202

**Comment by teamcons** [comment]
Hello
I assume you use 7.1 ?

Can you still use Plank normally ?
What functionality does this prevent ?
What did you change to cause this behaviour ?
How can we reproduce it ?

**Reply by alemstrom** [reply]
Stella/Charlie --  I don't know what plank is, when I type whatis plank at the command line it says stupidly simple.  I don't know the answers, its just what I am seeing in "journalctl -f" as I was curious what was causing my snapshot creation in 7.1...

**Reply by teamcons** [reply]
I need you to explain to me what you are trying to do, and the steps you did. There is little information in a pretty confused manner

Plank is the name of the dock, the horizontal list of icons on the bottom center of the screen, in elementary OS 5, 6, and 7

That line is an ssl error. It does not tell me what is going on: Is your dock crashing ? Is a functionality not working ? Are you trying to do something ?

**Reply by alemstrom** [reply]
Stella/Charlie - the plank is not modified.  I have not changed anything setting wise, it is not crashing, nothing appears to not be working - its functional as can be, just that error in journalctl -f is reporting that error.

To be honest, the error has disappeared. *shrug*

**Comment by teamcons** [comment]
Oh alright thanks
Yeah. It does not sound worrying in the slightest.

Some apps tend to spit errors or warnings in the background while
functioning normally. That is common

And this one is an old piece of software. It is replaced in next version.

alemstrom ***@***.***> schrieb am So., 8. Dez. 2024, 23:24:

**Comment by danirabbit** [comment]
Closing since it seems like this was an attempt at reporting an issue with Plank which is no longer shipped as of OS 8
